### DISCUSSION TITLE
ValueError: censoring survival function is zero at one or more time points

### DISCUSSION THREAD

**Discussion by saurabhbidwai** [sebp/scikit-survival#292] [Q&A]
I'm getting this error after changing my dependent variable i.e resp_days2, earlier its was working totally fine, can anyone help how to get a rid of it. Thanks in advance.

**Comment by kalkairis** [comment]
I resolved my issue with this error which was caused due to events that occurred in the final censoring time (e.g., I looked at a 5 year follow up and had a few individuals who had an event after exactly 5 years). Censoring/removing said individuals fixed the problem.

**Reply by sreens** [reply]
I faced this issue as well and the solution @kalkairis mentioned worked for me.  However, I am unclear why these kind of data points (events at  max observed time) should induce an error,@sebp can you please advise?
